app/experimenter/experiments/api/v4/views.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework.generics import (
+    ListAPIView,
+    RetrieveAPIView,
+)
+
+from experimenter.experiments.models import Experiment
+from experimenter.experiments.api.v4.serializers import (
+    ExperimentRapidSerializer,
+    ExperimentRapidRecipeSerializer,
+)
+
+
+class ExperimentListView(ListAPIView):
+    filter_fields = ("status",)
+    queryset = Experiment.objects.get_prefetched().filter(type=Experiment.TYPE_RAPID)
+    serializer_class = ExperimentRapidSerializer
+
+
+class ExperimentRapidDetailsView(RetrieveAPIView):
+    lookup_field = "slug"
+    queryset = Experiment.objects.get_prefetched()
```

Comment:
I'd filter this on type too, that way if someone puts in the wrong slug they'll get 404